rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -1226,6 +1226,9 @@ stream {
             proxy_buffering                         {{ $location.Proxy.ProxyBuffering }};
             proxy_buffer_size                       {{ $location.Proxy.BufferSize }};
             proxy_buffers                           {{ $location.Proxy.BuffersNumber }} {{ $location.Proxy.BufferSize }};
+            {{ if isValidByteSize $location.Proxy.ProxyMaxTempFileSize true }}
```

Comment:
`true` because this is an offset kind https://github.com/nginx/nginx/blob/master/src/http/modules/ngx_http_proxy_module.c#L579